src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -113,7 +115,7 @@ public void doGet_emptyDatabase_shouldReturnNoLecture() throws IOException {
   @Test
   public void doGet_oneLectureInDatabase_shouldReturnOneLecture() throws IOException {
     when(request.getParameter(LINK_INPUT)).thenReturn(TEST_LINK);
-    datastoreService.put(servlet.createLectureEntity(request));
+    datastoreService.put(LectureUtil.createEntity("", TEST_LINK, TEST_ID));
```

Comment:
Should I add inline comments here?

Reply:
you should always add inline comments. Especially for empty strings.

Reply:
yes please, they are always important for context.